index.html

```diff
@@ -39,13 +42,113 @@ <h1>Lofi girl</h1>
           </div>
          </div>
 
-         <div class="row" id="">
-
-          <div class="col"></div>
-
-         </div>
-
-         <!-- Footer-->
+         <!-- LIST -->
+         <div class="card" id="list">
+          <ul class="list-group list-group-flush">
+
+            <li class="list-group-item">
+              <div class="STATUS done">
+                <div class="row">
```

Comment:
Removing .STATUS .done as an overarching indication. Status needs to have its own section in the card that's visible both as colour & text indication eg. <p class=".status .done">Done</p> with matching CSS eg. .done {background: green}